fix(mnq): update quotas documentation (#3812)
scaleway/docs-content: identity-and-access-management/organizations-and-projects/additional-content/organization-quotas.mdx

| Resource                                       | Criteria   | Limits   | Scope            |
|------------------------------------------------|------------|----------|------------------|
| SQS queue storage                              | Max size   | 100 MB** | Scaleway Project |
| SQS maximum payload                            | Max size   | 256 KB   | SQS Queue        |
| Max number of SQS queues                       | Max number | 50       | Scaleway Project |
| Message age / retention period (SQS)           | Max        | 14 days  | Message          |
| Visibility timeout                             | Max        | 12 hours | SQS queue        |
| Max number of SNS topics                       | Max number | 50       | Scaleway Project |
| Max number of SNS subscribers per topic        | Max number | 50       | Scaleway Project |

> ** This limit applies to all your SQS queues withing your Scaleway project, it accounts for the total storage of all your project's queues.

Reviewer: ```suggestion
** This limit applies to all your SQS queues within your Scaleway project and covers the total storage used by all the queues in your project.
```

> ** This limit applies to all your SQS queues withing your Scaleway project, it accounts for the total storage of all your project's queues.

Reviewer 2: ```suggestion
** This limit applies to all your SQS queues within your Scaleway project and covers the total storage used by all the queues in your project.
```
